stock/6-dynamically-added-annotation (#28)
blacklabel/highcharts-fundamentals: highcharts-api/highcharts-stock/6-dynamically-added-annotation/README.md

# stock/6-dynamically-added-annotation

> Below the stock chart, create inputs that allow you to dynamically add crookedLine annotations. The annotation should be able to be modified after being added, so it will be necessary to add appropriate modules and a CSS file. Inputs should be limited to entering values from the chart, so after loading the chart you should add min and max attributes to the inputs extracted from the chart.

Reviewer: ```suggestion
Below the stock chart, create inputs that allow you to dynamically add `crookedLine` annotations. There should be a possibility to modify the annotation after adding it, so it will be necessary to add appropriate modules and a CSS file. Inputs should be limited to entering values from the chart, so after loading the chart you should add min and max attributes to the inputs extracted from the chart.
```
